.github/workflows/lint.yml

```diff
@@ -17,14 +17,14 @@ jobs:
   format:
     runs-on: ubuntu-latest
     steps:
+      - name: Clone repository
+        uses: actions/checkout@v4
+
       - name: Set up Go
         uses: actions/setup-go@v5
         with:
           go-version: stable
 
-      - name: Clone repository
-        uses: actions/checkout@v4
-
```

Comment:
![2025-04-02_10-05](https://github.com/user-attachments/assets/e70d1739-a5fe-452a-961f-85624c65193c)

This fixes that the CI cache isn't getting used because of the order of our clone and Go setup.